src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -314,6 +316,13 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
     validityIntervalStart && validityIntervalStart.toString()
   )
 
+  const prepareMetaDataHex = (metaData: Buffer | null): string | null => {
```

Comment:
Is there a trezor type we could define as return type of this function? Maybe not but if so it would be nice. 

Also we could have also the `prepareTtl` and `prepareValidityIntervalStart` return some `Trezor/Ledger` prefixed type so all the `prepare` function return either `Ledger` or `Trezor` type, if those types change we can always update the corresponding type-file and the ide would tell us what needs to be fixed. But I'd suggest addressing it in separate PR.